nss-day-cohort-45/C45-Website
# update grad date and cohort #

## Link to Ticket

Basic Info Update

## Description of Proposed Changes

Updated page to reflect our cohort # and Grad date

## Steps to Test

- [ ]  Make sure Nav Bar is Cohort 45, with correct Grad Date
- [ ] Make sure developer section has C45 with correct grad date
- [ ] Make sure footer is copyright 2021, with correct cohort

Added Headshots - make sure these pics are in images/classmates folder:

- [ ] Jess
- [ ] Ron
- [ ] Nick
- [ ] Meghan
- [ ] Pazia
- [ ] Steph
- [ ] Christine
- [ ] Jordan
- [ ] Chris D
- [ ] Tim
- [ ] Barry
- [ ] Greg
- [ ] Ryan D
- [ ] Tatiane


```sh
git fetch --all
git checkout jd-basicInfoUpdate

```
